pom.xml

```diff
@@ -80,8 +80,8 @@
 
     <gpg.skip>true</gpg.skip>
 
-    <bigdataoss.version>2.2.3-SNAPSHOT</bigdataoss.version>
-    <bigquery-connector.version>1.2.1-SNAPSHOT</bigquery-connector.version>
+    <bigdataoss.version>3.0.0-SNAPSHOT</bigdataoss.version>
+    <bigquery-connector.version>2.0.0-SNAPSHOT</bigquery-connector.version>
```

Comment:
Why are these properties not used in the other poms?

Reply:
Probably some legacy, we will clean up it in follow up.